Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -305,6 +334,47 @@ def runSelected(opt):
                       default=None,
                       action='store')
 
+    parser.add_option('--gpu',
+                      help='Use GPU workflow setup if available. It will force --requires-gpu required.',
+                      dest='gpuEnable',
+                      default=False,
+                      action='store_true')
+
+    parser.add_option('--requires-gpu',
+                      help='if GPU is required or not: forbidden (default, CPU-only), optional, required. For relvals, the GPU option will be turned off for optional.',
+                      dest='RequiresGPU',
+                      default='forbidden')
+
+    parser.add_option('--gpu-memory-mb',
+                      help='to specify GPU memory. Default = 8000 MB (for --requires-gpu = required).',
+                      dest='GPUMemoryMB',
+                      default=8000)
+
+    parser.add_option('--cuda-capabilities',
+                      help='to specify CUDA capabilities. Default = 6.0,6.1,6.2,7.0,7.2,7.5 (for RequiresGPU = required). Use comma to identify various CUDACapabilities',
+                      dest='CUDACapabilities',
+                      default='6.0,6.1,6.2,7.0,7.2,7.5')
+
+    parser.add_option('--cuda-runtime',
+                      help='to specify major and minor CUDA runtime used to build the application. Default = 11.2 (for RequiresGPU = required). FIX ME TO MATCH WITH CMSSW.',
+                      dest='CUDARuntime',
+                      default='11.2')
+
+    parser.add_option('--gpu-name',
+                      help='to specify GPU class. This is an optional parameter.',
+                      dest='GPUName',
+                      default='')
+
+    parser.add_option('--cuda-driver-version',
+                      help='to specify CUDA driver version. This is an optional parameter.',
+                      dest='CUDADriverVersion',
+                      default='')
+
+    parser.add_option('--cuda-runtime-version',
+                      help='to specify CUDA runtime version. This is an optional parameter.',
+                      dest='CUDARuntimeVersion',
+                      default='')
+
```

Comment:
What is the purpose of this parameter ?
To specify an exact match for the CUDA runtime, as opposed to `--cuda-runtime` which will match any runtime supported by the destination node ?

Reply:
I think CUDARuntimeVersion came with the proposal on what is optional parameters should be included. This PR follows the conclusion https://github.com/cms-sw/cmssw/pull/33057#issuecomment-808156471 and development in wmCore
https://github.com/dmwm/WMCore/issues/10388

We can drop it if it is not necessary. Thanks.

Reply:
OK, so the definition should still be the one I posted here: https://github.com/cms-sw/cmssw/pull/33057 .
I'll suggest a more descriptive help test.